Lib/python/pyrun.swg

```diff
@@ -299,10 +311,7 @@ SwigPyClientData_New(PyObject* obj)
       data->destroy = 0;
     }
     if (data->destroy) {
-      int flags;
-      Py_INCREF(data->destroy);
-      flags = PyCFunction_GET_FLAGS(data->destroy);
-      data->delargs = !(flags & (METH_O));
+      data->delargs = !(PyCFunction_GET_FLAGS(data->destroy) & METH_O);
```

Comment:
Just a note/observation that PyObject_GetAttrString is documented as returning a new reference, so the removal of the Py_INCREF is okay.

Reply:
Thank you for noting this - sorry if it wasn't immediately clear.